packages/fxa-auth-server/scripts/delete-unverified-accounts.ts

```diff
@@ -235,9 +235,19 @@ const init = async () => {
       return 0;
     }
 
-    const queue = new PQueue({ interval: 1000, intervalCap: taskLimit });
+    // Scaling suggestion is 500/5/50 rule, may start at 500/sec, and increase every 5 minutes by 50%.
+    // They also note increased latency may occur past 1000/sec, so we stop increasing as we approach that.
+    let lastScaleUp = Date.now();
+    let rateLimit = taskLimit;
+    let queue = new PQueue({ interval: 1000, intervalCap: rateLimit });
 
     for (const x of accounts) {
+      if (rateLimit < 950 && Date.now() - lastScaleUp > 6 * 60 * 1000) {
```

Comment:
Can we make `6` a variable called `scaleUpIntervalMins` or something similar?

Reply:
This is a hard limit from Google, so I don't think it needs to be configurable.

Reply:
Was an ask for readability - I'd have a hard time understanding why 6 is the magic number without the Slack thread, esp since it doesn't match the 5 in the comment above.

Non-blocking though, PR looks good